src/main/java/mate/academy/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -45,12 +45,18 @@ private String getErrorMessage(ObjectError error) {
         return error.getDefaultMessage();
     }
 
-    @ExceptionHandler(EntityNotFoundException.class)
-    protected ResponseEntity<Object> handleEntityNotFoundException(Exception ex) {
+    @ExceptionHandler({EntityNotFoundException.class, RegistrationException.class})
+    protected ResponseEntity<Object> handleException(Exception ex) {
+        HttpStatus status = (ex instanceof EntityNotFoundException)
+                ? HttpStatus.NOT_FOUND : HttpStatus.BAD_REQUEST;
+        return new ResponseEntity<>(getBody(ex, status), status);
+    }
+
+    private Map<String, Object> getBody(Exception ex, HttpStatus status) {
         Map<String, Object> body = new LinkedHashMap<>();
         body.put("timestamp", LocalDateTime.now());
-        body.put("status", HttpStatus.NOT_FOUND);
+        body.put("status", status);
         body.put("message", ex.getMessage());
-        return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
+        return body;
```

Comment:
writing response code TWICE seem inconvenient. Please create a method that will get all the required parameters and return a ResponseEntity object?